bin/drush-playwright-internal

```diff
@@ -39,7 +39,25 @@ if (!$test_id) {
   exit(1);
 }
 
-include_once '/var/www/html/web/core/includes/bootstrap.inc';
+// Get the docroot directory from the environment variable.
+$docroot_dir = getenv('DOCROOT');
+if (!$docroot_dir) {
+  print "DOCROOT must be defined as an environment variable.";
+  exit(1);
+}
+
+// Construct the path to the bootstrap.inc file based on the docroot directory.
+$bootstrap_path = "/var/www/html/$docroot_dir/core/includes/bootstrap.inc";
+
+// Check if the file exists and include it.
+if (file_exists($bootstrap_path)) {
+  include_once $bootstrap_path;
+}
+else {
+  print "bootstrap.inc file not found in the specified directory.";
```

Comment:
Even if we do this from composer.json, I like this. How about we print the bootstrap path too?

```suggestion
  print sprintf("bootstrap.inc file not found at %s", $bootstrap_path");
```